lib/rubocop/cop/layout/empty_comment.rb

```diff
@@ -5,7 +5,7 @@ module Cop
     module Layout
       # This cop checks empty comment.
       #
-      # @example
+      # @example AllowBorderComment: true (default)
```

Comment:
Maybe the examples that are always good and bad should be outside of the `AllowBorderComment` to reduce duplication and to make it more clear what this option controls.

Reply:
Sure! It makes the document clearer. I fixed it.